client/src/App.js

```diff
@@ -32,13 +39,13 @@ const StartingPoint: () => React$Node = () => {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+        <img src="" className="App-logo" alt="logo" />
```

Comment:
What's this change for?

Reply:
I planned to add an icon as logo there, but I haven't done it yet.